DoLahTest/ParserUnitTest.cpp

```diff
@@ -16,15 +16,26 @@
 using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 
 namespace DoLahTest {
-    TEST_CLASS(Tokenize) {
+    TEST_CLASS(Parser) {
+private:
+    std::string year;
+    std::string month;
+    std::string day;
 public:
     void parseAddTestMethod(std::string input, std::vector<std::string> expected) {
         try {
             std::vector<std::string> inputArr = DoLah::ParserLibrary::explode(input, " ");
-            std::vector<std::tm> datesVector = DoLah::TaskTokenizer::findDate(inputArr);
+            std::vector<std::tm> datesVector = DoLah::TaskTokenizer::findAndRemoveDate(inputArr);
             std::string dates = "";
             if (!datesVector.empty()) {
-                dates = DoLah::DateTimeParser::tmToString(datesVector.at(0));
+                for (size_t i = 0; i < datesVector.size(); i++) {
+                    dates += std::to_string(datesVector.at(i).tm_mday) + "/"
```

Comment:
so, this is just to parse from `tm` to `string`, correct? why not use boost? parsing became simple to understand.